### ISSUE TITLE
Error "You must enter a valid accessKeyId: Minimum length is 16" when attempting "amplify configure" from VS Code terminal

### ISSUE THREAD

**Issue by onerdy** [aws-amplify/amplify-cli#13028]
### Before opening, please confirm:

- [X] I have checked to see if my question is addressed in the [FAQ](https://github.com/aws-amplify/amplify-hosting/blob/master/FAQ.md).
- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-hosting/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting questions](https://github.com/aws-amplify/amplify-hosting/blob/master/CONTRIBUTING.md).
- [X] I have removed any sensitive information from my code snippets and submission.

### App Id

N/A

### AWS Region

us-west-1

### Amplify Hosting feature

Not Applicable

### Question

When I attempt to configure Amplify in VS Code using the command "amplify configure" I am able to successfully select my region, successfully login to AWS Management Console, successfully create a user and access key. However, when I paste in the accessKeyId, I receive the error message "You must enter a valid accessKeyId: Minimum length is 16" and I cannot get past this point. The Access Key is actually 20 characters so length does not seem to be the issue.

Does anyone have ideas about how to fix this?

Here is a log of terminal activity:

> 
> PS C:\repos\test> amplify configure
> Follow these steps to set up access to your AWS account:
> 
> Sign in to your AWS administrator account:
> https://console.aws.amazon.com/
> Press Enter to continue
> 
> Specify the AWS Region
> ? region:  us-west-1
> Follow the instructions at
> https://docs.amplify.aws/cli/start/install/#configure-the-amplify-cli
> 
> to complete the user creation in the AWS console
> https://console.aws.amazon.com/iamv2/home#/users/create
> Press Enter to continue
> 
> Enter the access key of the newly created user:
> ? accessKeyId:  <Access Key from AWS Management Console pasted here>
> >> You must enter a valid accessKeyId: Minimum length is 16

**Comment by ykethan** [comment]
Hey @onerdy, thank you for reaching out. when pasting in the `accessKeyId` does the Amplify CLI show a masked input?
for example: 
![image](https://github.com/aws-amplify/amplify-cli/assets/87995712/09df96c9-f4cf-4a3e-97ff-e25ed1d5db5a)

additionally, could you try pasting the `accessKeyId` in a text editor?

alternatively, we can directly modify the `~/.aws/credentials`  and `~/.aws/config` files to setup a profile as well. The run though the `amplify init` flow to setup a project.

**Comment by onerdy** [comment]
> 

Eureka!  I think you have solved my issue.

The accessKeyId field defaults to show the literal text of "[hidden]".  When I press CTRL+V intending to paste my access key, that default text disappears which made me think my pasted value is now in place (and just being hidden).  However, I see now that when I manually type text it shows asterisks as a mask.  Also, I have now noticed that if I right click in that field it properly pastes my access key from the clipboard.

So, the root of my issue was that when I pressed CTRL+V it was clearing that field rather than pasting in the key and I did not notice it since I expected the text to be hidden anyway.

Thank you for a quick reply.
